src/services/users/invite-user.test.js

```diff
@@ -0,0 +1,26 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../db')
+const inviteUser = require('./invite-user')
+
+
+describe('Invite a New User', () => {
+	let invitedUser
+	before(async () => {
+		invitedUser = {
+			email : faker.internet.email()
+		}
+	})
+	it('should invite a new user', async () => {
```

Comment:
similar to the `should sign in user`, this description is broad and not saying much

the scenarios to confirm are based on the code specification
1. an invite is recorded in the database
2. an email is sent to the email of the invitee
3. a response is returned

Currently, the test will still pass if we remove the part of the code that is sending of the email and we don't want that

So in this test,  we'll be confirming three things as separate test cases

1. whether the invite is recorded in the database
2. whether an email is sent to the user
3. whether the right response is returned from the service function


about confirming if an email is sent, we don't want a test to be sending actual emails as of now, so we might be using a concept called `mocking`

since we're using nodemailer, kindly do a bit of research on how to write tests for code which uses nodemailer to send emails

might join you on it later as you're finding that out

Reply:
alright, I'd do a lot of research for this. Thanks a lot

Reply:
should i take out the email being absent/invalid and user not being an admin?